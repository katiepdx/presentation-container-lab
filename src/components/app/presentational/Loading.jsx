// Loading screen component 
// Needs a snapshot test