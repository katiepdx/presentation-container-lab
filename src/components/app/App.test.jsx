import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Loading from './presentational/Loading';

describe('App component', () => {
  afterEach(() => cleanup());
  
  it('renders loading spinner', () => {
    const { asFragment } = render(<Loading />);
    expect(asFragment()).toMatchSnapshot();
  });
});
