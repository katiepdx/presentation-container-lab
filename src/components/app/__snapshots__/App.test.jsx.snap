// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders NewsSearch 1`] = `
<DocumentFragment>
  <div
    data-testid="display"
  >
    <h1>
      Welcome!
    </h1>
    <div>
      <label
        data-testid="Search"
      >
        Search
      </label>
      <input
        data-testid="search-input"
        id="text"
        name="text"
        type="text"
        value=""
      />
      <button
        data-testid="search-button"
      >
        Go!
      </button>
    </div>
  </div>
</DocumentFragment>
`;
